lib/orbit/operator_sign_in.ex

```diff
@@ -0,0 +1,35 @@
+defmodule Orbit.OperatorSignIn do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  alias Orbit.Employee
+  alias Orbit.RailLine
+  alias Orbit.SignInMethod
+  alias Orbit.User
+
+  @type t :: %__MODULE__{
+          signed_in_employee: Employee.t(),
```

Comment:
**suggestion (low)**: The `signed_in` prefixes here are kind of redundant.

Reply:
I wanted to make it clear exactly what the relationship between the `operator_sign_in` record and each party is, especially because in the future we might have individuals who both exist as entries in `users` and `employees`, which I believe is already the case for Glides. My feeling is that it may not be obvious to someone coming to this code for the first time that the "user" is the person doing the signing in and the "employee" is the person being signed in.

Reply:
Fair enough! Cannot argue with clarity 😃